api/seismic_site/settings.py

```diff
@@ -146,6 +147,9 @@ class Base(Configuration):
 
 class Dev(Base):
     DEBUG = True
+    ALLOWED_HOSTS = ["*"]
+    CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
This is fine for now, but it's recommended to replace it with specific hosts in production. 
`CORS_ORIGIN_ALLOW_ALL` should be replaced with a proper `CORS_ORIGIN_WHITELIST` config. See [this](https://github.com/adamchainz/django-cors-headers#cors_origin_whitelist) for reference.

Reply:
Not sure we'll use this feature. We might allow the API to be publicly accessed by anyone so that the data stays open. I'd like to establish this after a more thorough discussion.